404.php

```diff
@@ -19,7 +19,9 @@
 				<?php do_action('vantage_entry_main_top') ?>
 
 				<header class="entry-header">
-					<h1 class="entry-title"><?php echo apply_filters( 'vantage_404_title', __( "That page can't be found.", 'vantage' ) ); ?></h1>
+					<?php if( siteorigin_page_setting( 'page_title' ) ) { ?>
```

Comment:
In templates, we should try stick to the convention of using the alternative control syntax. This syntax is easier to read in files that combine a lot of HTML and PHP. It's also the convention used by _s.

So this would be

```
<?php if( siteorigin_page_setting( 'page_title' ) ) : ?>
    <h1 class="entry-title"><?php echo apply_filters( 'vantage_404_title', __( "That page can't be found.", 'vantage' ) ); ?></h1>
<?php endif; ?>
```